app/controllers/concerns/discovery.rb

```diff
@@ -18,7 +19,8 @@ def discovery
           cloud_jobs_failed:                 SaltJob.failed.count,
           admin:                             Minion.find_by(minion_id: "admin"),
           retryable_bootstrap_orchestration: Orchestration.retryable?(kind: :bootstrap),
-          retryable_upgrade_orchestration:   Orchestration.retryable?(kind: :upgrade)
+          retryable_upgrade_orchestration:   Orchestration.retryable?(kind: :upgrade),
+          last_orchestration_at:             last_orchestration_at
```

Comment:
this could be a one liner, and no need to cache it in a variable

```ruby
Orchestration.try(:last).try(:created_at)
```

Reply:
Forgot the `try` method. If we were using ruby 2.5 we could use `.&`. I'll fix that. Thanks!